### ISSUE TITLE
Latitude is cleared before it can be saved

### ISSUE THREAD

**Issue by TWCunningham** [nemethviktor/GeoTagNinja#1]
Windows 11, English

When I paste in a latitude value, it stays long enough for me to get the reverse geocode information, but when I press OK it shows up in the main window as zero.

**Comment by nemethviktor** [comment]
Hi, thanks for that, can you be a little more specific as to where you're pasting the data (a screenshot video would be the best but if not then just a bit more detail.) If I paste a value from the clipboard in the Edit Data's Latitude field it works as expected. 
Or do you mean "paste" as in the get-data-from-other-file?

**Comment by TWCunningham** [comment]
Well of course, I can't duplicate the behavior now, but I thought this was interesting:

![GeoTagNinja01](https://user-images.githubusercontent.com/11800584/186045962-1bb2c407-af36-4bdd-a4b5-31a7cb261306.jpg)

I click on the highlighted line and in the Edit Data box the latitude and longitude are zero.  In particular this happens after I have copied the GPS data from another photo but haven't saved the data yet.

**Comment by TWCunningham** [comment]
Hang on a sec, I think I've got it (I tried using the XBox GameBar to record the screen, but it doesn't seem to capture the dialog box, so here are a sequence of screenshots):

![GeoTagNinja02](https://user-images.githubusercontent.com/11800584/186049004-e76ae43c-a941-4b31-a6c4-6c0a1803a0cf.jpg)

So here you have the latitude zeroed out, and the map is somewhere in Columbia.  In the next shot, I change the latitude to where the picture actually is, New Hampshire:

![GeoTagNinja03](https://user-images.githubusercontent.com/11800584/186050322-2d58704f-f3ad-4c3a-8d08-548a32660353.jpg)

I press "Get From Web" under Location Data; it updates the fields and tells me the data has been updated:

![GeoTagNinja04](https://user-images.githubusercontent.com/11800584/186052982-bb6edf33-876e-44e9-a15b-9b0a576d6723.jpg)

I press OK to dismiss the alert, then OK in the dialog box.  Et voila!  We are back in Columbia again:

![GeoTagNinja05](https://user-images.githubusercontent.com/11800584/186053650-107cb79e-30ec-443f-b028-9d590e69ef66.jpg)
